CHANGELOG.md

```diff
@@ -75,6 +75,11 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#2395](https://github.com/realm/SwiftLint/issues/2395)
 
+* Specify what type of compiler protocol initializer violated the
+  `compiler_protocol_init` rule.
```

Comment:
requires two trailing spaces as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes

Reply:
fixed